contracts/0.4.24/nos/NodeOperatorsRegistry.sol

```diff
@@ -386,7 +388,7 @@ contract NodeOperatorsRegistry is INodeOperatorsRegistry, IsContract, AragonApp
 
             effectiveStake = operator.usedSigningKeys.sub(operator.stoppedValidators);
             uint256 reward = uint256(effectiveStake).mul(_totalReward).div(uint256(effectiveStakeTotal));
-            require(IERC20(_token).transfer(operator.rewardAddress, reward), "TRANSFER_FAILED");
+            require(ERC20(_token).safeTransfer(operator.rewardAddress, reward), "TRANSFER_FAILED");
```

Comment:
move the proposal to remove `_token` from `distributeRewards()` args to the separate issue. This is not related to coverage.